aipcli/config.go

```diff
@@ -13,22 +14,22 @@ import (
 // Config is the configuration for the protoc-gen-go-aip-cli plugin.
 type Config struct {
 	// Hosts is a map from host ID to host address.
+	// In order for SagaAuth to switch to dev token, in dev env the host ID should start with 'dev-'
 	Hosts map[string]string
 	// DefaultHost is the host ID of the default host.
 	DefaultHost string
 	// Root is the name of the root module package.
 	Root string
 	// GoogleCloudIdentityTokens indicates if Google Cloud Identity Tokens should be automatically generated.
 	GoogleCloudIdentityTokens bool
-	// Uses in a predetermined file relative to local config path for the Identity Token.
-	CachedIdentityTokenPath string
 }
 
 const (
-	addressFlag  = "address"
-	tokenFlag    = "token"
-	insecureFlag = "insecure"
-	verboseFlag  = "verbose"
+	addressFlag     = "address"
+	tokenFlag       = "token"
+	insecureFlag    = "insecure"
+	verboseFlag     = "verbose"
+	hostIDDevPrefix = "dev-"
```

Comment:
We shouldn't add any concepts of environment as it may differ from system to system. The name of the environment can be called anything and may not have any dev or prod